plugins/all-modules-page/out/index.md

```diff
@@ -0,0 +1,10 @@
+/
+
+# Sample project
+
+Sample documentation with [external link](https://www.google.pl)
+
+## All modules:
+
+| Name |
+|---|
```

Comment:
This keeps appearing after building dokka, very annoying, we need to fix this :) Can you remove it from the PR?